.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["env"]
```

Comment:
What does this do?

Reply:
Sets the compatibility level that babel is aiming for based on the installed version of node

Reply:
Or, "sourcing from" rather than "aiming for". Means that babel will intend to support any JS features supported by your installed version of node.